runtime/core/speaker/e2e_speaker.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2023 Chengdong Liang (liangchengdong@mail.nwpu.edu.cn)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SPEAKER_E2E_SPEAKER_H_
+#define SPEAKER_E2E_SPEAKER_H_
+
+#include <string>
+#include <vector>
+#include <memory>
+
+#include "frontend/feature_pipeline.h"
+#include "speaker/speaker_model.h"
+
+
+namespace wespeaker {
+
+class E2ESPEAKER {
```

Comment:
E2eSpeaker, the name convention should be consistent to global code.

Reply:
Done.